include/bluetooth/bluetooth.h

```diff
@@ -166,6 +166,34 @@ int bt_set_name(const char *name);
  */
 const char *bt_get_name(void);
 
+/**
+ * @brief Get local Bluetooth appearance
+ *
+ * Bluetooth Appearance is a description of the external appearance of a device
+ * in terms of an Appearance Value.
+ *
+ * @see https://specificationrefs.bluetooth.com/assigned-values/Appearance%20Values.pdf
+ *
+ * @returns Appearance Value of local Bluetooth host.
+ */
+uint16_t bt_get_appearance(void);
+
+/**
+ * @brief Set local Bluetooth appearance (persistently)
+ *
+ * Automatically preserves the new appearance across reboots if
+ * @kconfig{CONFIG_BT_SETTINGS} is enabled.
+ *
+ * This symbol not linkable if @kconfig{CONFIG_BT_DEVICE_APPEARANCE_DYNAMIC} is
+ * not enabled.
```

Comment:
Avoid double negation

```suggestion
 * This symbol is linkable if @kconfig{CONFIG_BT_DEVICE_APPEARANCE_DYNAMIC} is
 * enabled.
```